modules/admin/con-rhdh-deployment-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+// assembly-rhdh-operator-deployment-config
+
+[id="con-rhdh-deployment-config_{context}"]
+= {product} deployment configuration
+
+As of 1.3, the {product} operator exposes a new `rhdh.redhat.com/v1alpha2` API Version of its Custom Resource Definition (CRD). This CRD exposes a generic `spec.deployment.patch` field, which gives you full control over the RHDH Deployment resource. This field can be a fragment of the standard `apps.Deployment` Kubernetes object.
```

Comment:
```suggestion
The {product} operator exposes a new `rhdh.redhat.com/v1alpha2` API Version of its Custom Resource Definition (CRD). This CRD exposes a generic `spec.deployment.patch` field, which gives you full control over the RHDH Deployment resource. This field can be a fragment of the standard `apps.Deployment` Kubernetes object.
```

Wondering if we need the version here. This will be part of the docs for 1.3+ anyway, so not included in prior versions. WDYT?

Reply:
Hrm...if this isn't back ported to v1.2, and/or v.1.1, it might be beneficial for the user to know this has only been available starting with 1.3, but also, I don't know enough about the user base yet to know if they always update, and don't know enough about the product to know if it always automatically updates and there isn't a way for users to continue remaining on 1.1 or 1.2.

However, mentioning this in 1.3 Release Notes, and noting that this is available in 1.3+, makes sense. So including 1.3 here won't be needed. OK. Will make the change. Thanks!

Reply:
Yeah, this should definitely not be backported to 1.2 or 1.1, because the operator in these previous versions will not handle this new `v1alpha2` API version in the Custom Resource.
As I understand the docs on docs.redhat.com, users need to pick a version of the product anyway: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2 , so they should only see this in 1.3+

> However, mentioning this in 1.3 Release Notes, and noting that this is available in 1.3+, makes sense.

Yes, the engineering JIRA issue that added this should normally have the Release Notes field filled. cc @gazarenkov

Reply:
Yep, users can pick a version of the product and see docs specific to that version. Docs can be cherry-picked back to previous versions if whatever is being documented also applies to previous versions.

Appreciate the clarification that this will not be back ported!